torch/csrc/jit/autodiff.cpp

```diff
@@ -7,6 +7,15 @@ namespace torch { namespace jit {
 
 using value_list = std::vector<Value*>;
 
+
+std::unordered_set<Symbol> differentiable_kinds = {
```

Comment:
Can we add a note to the switch statement saying you also need to update differentiable_kinds?